pkg/grapicmd/internal/usecase/initialize_project_usecase.go

```diff
@@ -132,7 +132,7 @@ func (u *initializeProjectUsecase) InstallDeps(rootDir string, cfg InitConfig) e
 				return errors.WithStack(err)
 			}
 		}
-		err := invoke("go", "get", "./...")
+		err := invoke("go", "mod", "tidy")
```

Comment:
I couldn't find the source, but what I want to do is update go.mod, so I'm committing it thinking that what I'm doing shouldn't change.